worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/FAWE_Spigot_v1_15_R2.java

```diff
@@ -503,4 +415,392 @@ public int getInternalBiomeId(BiomeType biome) {
         BiomeBase base = CraftBlock.biomeToBiomeBase(BukkitAdapter.adapt(biome));
         return IRegistry.BIOME.a(base);
     }
+    
+    private static class ReneratorImpl extends Regenerator<IChunkAccess, ProtoChunk, Chunk, ReneratorImpl.ChunkStatusWrap> {
```

Comment:
I think a separate package for these impl classes would be better than nesting them all inside their respective adapter impls.

"adapter/impl/regen" or so

Reply:
Agreed.